src/pyFAI/resources/gui/detector-selection-drop.ui

```diff
@@ -157,55 +150,51 @@ background-color: transparent;
          </sizepolicy>
         </property>
         <property name="title">
-         <string/>
+         <string>Sensor:</string>
```

Comment:
```suggestion
         <string>Sensor</string>
```

Reply:
> Why removing the colon ? Orientation has also one. should it also be removed ?
Because it's a group box. If orientation is also a group box, there shouldn't be a colon as well.

Other group boxes do not have the colon in the screen shot: https://github.com/silx-kit/pyFAI/pull/2627#issuecomment-3474077386